server.js

```diff
@@ -86,21 +97,30 @@ var createAndLogEvent = function(data, req) {
     var value = parseValue(properties[key]);
     entry += ' ' + key + '=' + value;
   }
-  entry += '\n';
-  fs.appendFile(path.resolve(__dirname, './events.log'), entry, function(err) {
-    if (err) {
-      console.log(err);
-    } else {
-      //console.log("Logged tracked data");
-    }
-  });
+  if (argv.docker) {
+    console.info(entry);
+  } else {
+    entry += '\n';
+    fs.appendFile(path.resolve(__dirname, './events.log'), entry, function(err) {
+      if (err) {
+        console.error(err);
+      } else if (argv.debug) {
+        console.log('Logged tracked data');
+      }
+    });
+  }
 };
 
 /*
  * Use Middlewares
  **********************************
  */
-app.use(express.logger());
+if (argv.docker) {
+  var morgan = require('morgan');
+  app.use(morgan('combined', { stream: process.stderr }));
```

Comment:
Cool, so I'll get this merged into master tomorrow which should make it shippable in Docker unless we need to doing anything for SSL support. Right now it just has some dummy certs for testing.